manifests/crd.yaml

```diff
@@ -3,139 +3,138 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes/enhancements/pull/1870
-    controller-gen.kubebuilder.io/version: v0.6.0
+    controller-gen.kubebuilder.io/version: v0.7.0
   creationTimestamp: null
   name: noderesourcetopologies.topology.node.k8s.io
-  namespace: ""
 spec:
   group: topology.node.k8s.io
   names:
     kind: NodeResourceTopology
     listKind: NodeResourceTopologyList
     plural: noderesourcetopologies
     shortNames:
-      - node-res-topo
+    - node-res-topo
     singular: noderesourcetopology
-  scope: Namespaced
+  scope: Cluster
```

Comment:
I assume this is supported or taken into account in other parts of the code

Reply:
yes, this is what the update is about